secure-api-gateway-ob-uk-rs-validation/secure-api-gateway-ob-uk-rs-validation-obie/src/test/java/com/forgerock/sapi/gateway/ob/uk/rs/validation/obie/v4/payment/OBWriteDomestic2DataInitiationInstructedAmountValidatorTest.java

```diff
@@ -53,7 +54,7 @@ public void failsValidationWhenInstructedAmountCurrencyInvalid() {
 
     @Test
     public void failsValidationWhenInstructedAmountLessThanOrEqualToZero() {
-        final String[] invalidAmounts = new String[] {
+        final String[] invalidAmounts = new String[]{
```

Comment:
Note you can have explicit paramterized tests, so the test is easy and values supplied as args:
https://www.baeldung.com/parameterized-tests-junit-5

Reply:
Thanks!